.github/actions/spelling/expect.txt

```diff
@@ -341,4 +347,5 @@ Wunused
 WZDNCRFJ
 XPLATSTR
 xsi
-zy
+Zanollo
+zy
```

Comment:
Git/Diff tools/Unix tools in general appreciate having a new line at the end of files. (You can take the line that's currently being inserted at the beginning of the file 😉.)

(The tool itself doesn't care much these days, but the next person to touch the file will probably be blamed for the last line in the file, just as you're currently taking blame for `zy` which was added by someone else.)

n.b.. in this case, that's going to be short-lived anyway, I need to send a PR to upgrade check-spelling, at which point two-letter words will be ignored entirely so they'll be dropped no later than that PR.

Reply:
Sorry, since the file was alphabetically ordered I thought it needed to be that way. Will have that into account for next time.

Reply:
It wants to be sorted, yes. It doesn't need to be (I understand that people don't like running arbitrary code, but are willing to make hand edits). That said, sorting it is generally appreciated. (At the very least, sorting makes it easier for a reader to spot related items with some typo.)

The commands it suggests will clean out stale entries, order it, and ensure a new line at the of file.

It's ok to do any/all of these by hand.

I have a feature that should be generally usable soon that will let you ask the action to do this cleanup instead of doing it by hand, but I'm letting it bake with some other almost done features. Hoping to ship it soon.

Reply:
@jsorf, I hear lots of complaints from people until the dictionary gets cleaned up a "bit". It's an awesome tool after one has adapted! In the winget-pkgs directory we will have plenty of manifests! Is there a way to ignore files by extension, or better yet, support localization "to some extent"?

Reply:
@denelon: `excludes.txt` should let you exclude things fairly easily. Certainly, ignoring files by extension or path (or both).

Also, there's an optional advice file which would help:

https://github.com/check-spelling/spell-check-this/blob/main/.github/actions/spelling/advice.md

(It's content is configurable, this is just a sample. Note that in older versions of the action the file needs to be `.txt`, so test, ...)

I'm generally including it when I send PRs to update the action.